tests/unit/client-eviction.tcl

```diff
@@ -0,0 +1,475 @@
+tags {"external:skip"} {
+
+# Get info about a redis client connection:
+# name - name of client we want to query
+# f - field name from "CLIENT LIST" we want to get
+proc client_field {name f} {
+    set clients [split [string trim [r client list]] "\r\n"]
+    set c [lsearch -inline $clients *name=$name*]
+    if {![regexp $f=(\[a-zA-Z0-9-\]+) $c - res]} {
+        throw no-client "no client named $name found with field $f"
```

Comment:
```suggestion
        error "no client named $name found with field $f"
```
`thow` is supported after `tcl8.6`, whereas we require `tcl8.5`.

Reply:
thanks, fixed.